### ISSUE TITLE
 Allow single-line declaration of indexed arrays in Squiz.Arrays.ArrayDeclaration.SingleLineNotAllowed

### ISSUE THREAD

**Issue by Slawutich** [squizlabs/PHP_CodeSniffer#1730]
I suggest adding a property "allowSingleLineIndexedArrays" to the rule Squiz.Arrays.ArrayDeclaration.SingleLineNotAllowed

Config:
```xml
<rule ref="Squiz.Arrays.ArrayDeclaration">
    <properties>
        <property name="allowSingleLineIndexedArrays" value="true" />
    </properties>
    <exclude name="Squiz.Arrays.ArrayDeclaration.ValueNoNewline" />
</rule>
```
PHP code:
```php
// Good
$var = array('value1', 'value2');

// Good
$var = array(
        'value11', 'value12',
        'value21', 'value22',
       );

// Bad
$var = array('key1' => 'value1', 'key2' => 'value2');

$var = array(
        // Bad
        'key1' => array('key11' => 'value11', 'key12' => 'value12'),
        // Good
        'key2' => array('value21', 'value22'),
        'key3' => 'value3',
       );
```

I would like to provide control over associative arrays and weaken rules on indexed arrays.
Maybe there is a way, without adding a property?

**Comment by jrfnl** [comment]
Related to #582

**Comment by gsherwood** [comment]
Closing due to lack of additional interest
